changelog/_unreleased/2021-05-28-symfony-command-constants.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Symfony command constants  
+issue: NONE  
+author: Ben Pesch  
+author_email: b.pesch@protonmail.com   
+author_github: bpesch
+---
+# Core
+* Used Symfony command constants instead of integers for command status
```

Comment:
Please make use of the [keywords](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md). (e.g. Changed returns to use Symfony command constants instead of integers for command status)

Reply:
Updated!